theme/templates/pages/group.html

```diff
@@ -58,6 +58,51 @@ <h2 class="group-title">{{ group.name }}</h2>
                 <h4 class="text-muted">{{ group.gaccess.purpose|linebreaks }}</h4>
 
                 <p>{{ group.gaccess.description|linebreaks }}</p>
+                {% if not profile_user.is_group_member and profile_user.is_authenticated %}
```

Comment:
Making a note here to remember to update conditionals accordingly in work done in #4372